src/scale/scale.rs

```diff
@@ -29,12 +29,23 @@ pub struct Scale {
 }
 
 impl Scale {
-    /// Create a new scale.
+    /// Create a new ascending scale.
     pub fn new(
         scale_type: ScaleType,
         tonic: PitchClass,
         octave: u8,
         mode: Option<Mode>,
+    ) -> Result<Self, ScaleError> {
+        Self::new_in_direction(scale_type, tonic, octave, mode, Direction::Ascending)
+    }
+
+    /// Create a new scale with a given direction.
+    pub fn new_in_direction(
```

Comment:
Suggestion: now the Direction support is being added, i think we can add the direction parameter to `Scale::new method` and remove `new_in_direction`.  Why should we have two functions here?

Reply:
I agree, I was just trying to preserve the original API. I'll go through and refactor to a single unified `new` when I get a chance.

Reply:
Cool! Yes, I think we will need to bump at least the minor version of the crate after this change.